src/addrman.h

```diff
@@ -537,12 +552,14 @@ class CAddrMan
     }
 
     //! Mark an entry as accessible.
-    void Good(const CService &addr, int64_t nTime = GetAdjustedTime())
+    void Good(const CService &addr, bool test_before_evict = true, int64_t time = GetAdjustedTime())
     {
-        LOCK(cs);
-        Check();
-        Good_(addr, nTime);
-        Check();
+        {
```

Comment:
Why the extra `{}`?

Reply:
Not sure, Bitcoin used to have a second set of braces there when this pull request was originally created. They have been removed in the latest version.

https://github.com/bitcoin/bitcoin/blob/fb26bf0ea3822638b10a783f054c280fc053a2b5/src/addrman.h#L526